locales/ca_ES/LC_MESSAGES/messages.po

```diff
@@ -790,3 +790,147 @@ msgstr ""
 
 msgid "Contrôle de vitesse par balises (KVB)"
 msgstr ""
+
+msgid "1050mm"
+msgstr ""
+
+msgid "762mm"
+msgstr ""
+
+#, fuzzy
+msgid "Broad gauge"
+msgstr "Gàlib"
+
+msgid "1668mm"
+msgstr ""
+
+#, fuzzy
+msgid "Dual gauge"
+msgstr "Gàlib"
```

Comment:
Do "broad gauge" and "dual gauge" have the same translation?

Same for other locales, e.g. FI, FR, HU.

Reply:
msgmerge tried to be clever and added some guesses, I added a commit to stop this.